src/glue/connection.ts

```diff
@@ -16,27 +16,88 @@ export enum ConnectionType {
  * Configuration for the Glue Workflow resource.
  */
 export interface ConnectionProps extends ResourceProps {
+  /**
+   * The type of the connection. Currently, these types are supported:
+   *    JDBC - Designates a connection to a database through Java Database Connectivity (JDBC).
+   *    KAFKA - Designates a connection to an Apache Kafka streaming platform.
+   *    MONGODB - Designates a connection to a MongoDB document database.
+   *    NETWORK - Designates a network connection to a data source within an Amazon Virtual Private Cloud environment (Amazon VPC).
+   * 
+   * @see [AWS::Glue::Connection ConnectionInput](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-connection-connectioninput.html#cfn-glue-connection-connectioninput-connectiontype)
+   */
   readonly connectionType: ConnectionType;
+  /**
+   * A description for the Connection.
+   */
   readonly description?: string;
+  /**
+   * The name of the connection. Connection will not function as expected without a name.
+   * 
+   * @see [AWS::Glue::Connection ConnectionInput](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-connection-connectioninput.html#cfn-glue-connection-connectioninput-name)
+   */
   readonly name?: string;
+  /**
+   * List of Key/Value pairs defining the properties of the Connection 
```

Comment:
Would recommend removing the available properties as it is left generic to ensure forward compatibility.

Reply:
Removed the available properties portion.